packages/codegen/src/codegen/.hygen/templates/powerhouse/generate-document-model-module/customTest.esm.t

```diff
@@ -24,7 +30,13 @@ describe('<%= h.changeCase.pascal(module) %> Operations', () => {
 
 <% actions.forEach(action => { _%>
     it('should handle <%= h.changeCase.camel(action.name) %> operation', () => {
-        const input = generateMock(z.<%= h.changeCase.pascal(action.name) %>InputSchema());
+        // generate a random id
+        // const id = documentModelUtils.hashKey();
```

Comment:
can be removed?

Reply:
I can remove it, but I putted this one here so when we generate the test file and the editor, you have an example of how to generate a random ID.